React/Base/RCTConvert.m

```diff
@@ -229,11 +238,10 @@ + (NSDate *)NSDate:(id)json
   @"dashed": @(NSUnderlinePatternDash | NSUnderlineStyleSingle),
 }), NSUnderlineStyleSingle, integerValue)
 
-RCT_ENUM_CONVERTER(RCTTextDecorationLineType, (@{
+RCT_MULTI_ENUM_CONVERTER(RCTTextDecorationLineType, (@{
   @"none": @(RCTTextDecorationLineTypeNone),
   @"underline": @(RCTTextDecorationLineTypeUnderline),
-  @"line-through": @(RCTTextDecorationLineTypeStrikethrough),
-  @"underline line-through": @(RCTTextDecorationLineTypeUnderlineStrikethrough),
+  @"line-through": @(RCTTextDecorationLineTypeStrikethrough)
```

Comment:
style: trailing comma after the last item in dictionaries (and arrays for that matter) when the language allows